config/routes.rb

```diff
@@ -1,12 +1,20 @@
 Rails.application.routes.draw do
   root   'products#index'
+
+  get    'password_resets/new'
+  get    'password_resets/edit'
+
   get    '/login',                    to: 'sessions#new'
   post   '/login',                    to: 'sessions#create'
   delete '/logout',                   to: 'sessions#destroy'
   get    '/signup',                   to: 'users#new'
+
   get    '/dashboard',                to: 'dashboard#index'
   post   '/add_to_cart/:product_id',  to: 'carts#add_to_cart', as: 'add_to_cart'
   get    '/cart',                     to: 'carts#index'
+  
   resources :products
   resources :users
+  resources :account_activations, only: [:edit]
+  resources :password_resets,     only: [:new, :create, :edit, :update]
```

Comment:
when you have only 1 method, you do not need the `[]`, and when you have multiple, you could use `%i[new create etc.]`

Reply:
Thanks!